schematics/web/authentication/files/src/routing/routes/SignIn/SignInView/SignInView.tsx.template

```diff
@@ -29,12 +30,13 @@ export const SignInView: VFC<Props> = ({ error, loading, onSignIn }) => {
             Sign in
           </Typography>
           <SignInForm error={error} loading={loading} onSubmit={onSignIn} />
-          <Link to={Paths.FORGOT_PASSWORD_PATH} underline="none">
+          <Link to={Paths.FORGOT_PASSWORD_PATH} underline="none" data-testid="forgot-password-button">
```

Comment:
I think prettier should detect this line as an issue in the terminal because there are to much signs in one line, isn't it?